test/bdd/pkg/bootstrap/steps.go

```diff
@@ -14,13 +14,13 @@ import (
 	"github.com/cucumber/godog"
 	"github.com/google/uuid"
 
-	"github.com/trustbloc/hub-auth/pkg/restapi/operation"
-	bddctx "github.com/trustbloc/hub-auth/test/bdd/pkg/context"
-	"github.com/trustbloc/hub-auth/test/bdd/pkg/login"
+	"github.com/trustbloc/auth/pkg/restapi/operation"
+	bddctx "github.com/trustbloc/auth/test/bdd/pkg/context"
+	"github.com/trustbloc/auth/test/bdd/pkg/login"
 )
 
 const (
-	bootstrapDataPath = login.HUB_AUTH_HOST + "/bootstrap"
+	bootstrapDataPath = login.AUTH__HOST + "/bootstrap"
```

Comment:
And here?

Reply:
[fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)